lib/ErrorResponse.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+
+const STBase = require('./STBase');
+
+class ErrorResponse extends STBase {
```

Comment:
What's the intention for this class since it does exactly what STBase does?

Reply:
It seemed odd to me to be instantiating the same class for an error response as for an empty response that was not an error. This code from the existing `lambda.js` implementation also seems confusing to me:
```
const ErrorResponse = require('./STBase');
```
My thought was that even though the classes are the same now, we might have some reason to make them different in the future, and it would be a less disruptive change. I thought about removing `setError()` from `STBase` and having it only be in `ErrorResponse`, and perhaps even include it in the constructor, but got lazy and didn't do that because it would have broken some tests, which I admit is a lame reason :-). 

I'm not wedded to having the separate class, but the fact that STBase was renamed in lambda.js, presumably for clarity in that context, seems to argue for a separate class to me.

Reply:
Ah, nevermind. Looking at it again I think I'll just stick with STBase, and perhaps rename the constant in lambda.js